dlpy/network.py

```diff
@@ -1638,12 +1638,14 @@ def extract_conv_layer(layer_table):
     else:
         conv_layer_config['include_bias'] = True
 
-    padding_width = dl_numval[layer_table['_DLKey1_'] == 'convopts.pad_left'].tolist()[0]
-    padding_height = dl_numval[layer_table['_DLKey1_'] == 'convopts.pad_top'].tolist()[0]
-    if padding_width != -1:
-        conv_layer_config['padding_width'] = padding_width
-    if padding_height != -1:
-        conv_layer_config['padding_height'] = padding_height
+    # pad_top and pad_left are added after vb015
+    if 'convopts.pad_left' in layer_table['_DLKey1_'].values and 'convopts.pad_top' in layer_table['_DLKey1_'].values:
+        padding_width = dl_numval[layer_table['_DLKey1_'] == 'convopts.pad_left'].tolist()[0]
+        padding_height = dl_numval[layer_table['_DLKey1_'] == 'convopts.pad_top'].tolist()[0]
+        if padding_width != -1:
+            conv_layer_config['padding_width'] = padding_width
+        if padding_height != -1:
+            conv_layer_config['padding_height'] = padding_height
```

Comment:
we are also handling the "else" case right?

Reply:
When it is -1, it means this option is not specified. This is used to keep back compatible.

Reply:
Since the minimum value of padding_width and padding_height are 0 and -1 means the option is not specified, it has the following scenarios:
1. padding_width or padding_height not specified in the layer option: variables padding_width=-1 or padding_height=-1 and the according layer options are not specified.
2. padding_width or padding_height specified in the layer option: variables padding_width>=0 or padding_height>=0 and the according layer options are equal to the variables. 
So, I don't think we need to handle else case.